src/openlcb/MultiConfiguredPC.hxx

```diff
@@ -138,10 +138,20 @@ public:
         ConfigUpdateService::instance()->register_update_listener(this);
         producedEvents_ = new EventId[size * 2];
         std::allocator<debouncer_type> alloc;
+// C++20 standard removed the construct and destruct methods from
```

Comment:
this comment is not needed to be added here; let's put it into the PR description and git log will remember it for us.